app/src/main/java/org/mozilla/webmaker/activity/Play.java

```diff
@@ -15,10 +19,23 @@ public Play() {
         super("project", R.id.page_layout, R.layout.page_layout, R.menu.menu_play);
     }
 
+    protected String login_username;
+    protected String app_creator;
```

Comment:
Probably best to be consistent about how we reference the "app_creator" and project which would be: `projectAuthor` and `projectId`.